front/src/reducers/osrdconf/stdcmConf/index.ts

```diff
@@ -1,19 +1,25 @@
+import { infraState, buildInfraStateReducers, InfraStateType } from 'reducers/infra';
```

Comment:
To stick with front syntax recommandation, could you import types like this (I am and will probably miss other imports like that)
```suggestion
import { infraState, buildInfraStateReducers } from 'reducers/infra';
import type { InfraStateType } from 'reducers/infra';
```

Reply:
Done